src/chrome/content/rules/GoDaddy.xml

```diff
@@ -1,22 +1,4 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Non-2xx HTTP code: http://ca.m.godaddy.com/ (200) => https://ca.m.godaddy.com/ (403)
-Non-2xx HTTP code: http://sg.m.godaddy.com/ (200) => https://sg.m.godaddy.com/ (403)
-Non-2xx HTTP code: http://godaddy.com/ (200) => https://godaddy.com/ (403)
-Fetch error: http://community.godaddy.com/ => https://community.godaddy.com/: (51, "SSL: no alternative certificate subject name matches target host name 'community.godaddy.com'")
-Fetch error: http://support.godaddy.com/ => https://support.godaddy.com/: (60, 'SSL certificate problem: unable to get local issuer certificate')
-Non-2xx HTTP code: http://supportcenter.godaddy.com/ (200) => https://supportcenter.godaddy.com/ (403)
-Non-2xx HTTP code: http://au.godaddy.com/ (200) => https://au.godaddy.com/ (403)
-Non-2xx HTTP code: http://dk.godaddy.com/ (200) => https://dk.godaddy.com/ (403)
-Non-2xx HTTP code: http://fr.godaddy.com/ (200) => https://fr.godaddy.com/ (403)
-Non-2xx HTTP code: http://it.godaddy.com/ (200) => https://it.godaddy.com/ (403)
-Non-2xx HTTP code: http://no.godaddy.com/ (200) => https://no.godaddy.com/ (403)
-Non-2xx HTTP code: http://pk.godaddy.com/ (200) => https://pk.godaddy.com/ (403)
-Non-2xx HTTP code: http://pt.godaddy.com/ (200) => https://pt.godaddy.com/ (403)
-Non-2xx HTTP code: http://sg.godaddy.com/ (200) => https://sg.godaddy.com/ (403)
-Non-2xx HTTP code: http://fi.godaddy.com/ (200) => https://fi.godaddy.com/ (403)
-Non-2xx HTTP code: http://tr.godaddy.com/ (200) => https://tr.godaddy.com/ (403)
```

Comment:
Basically all of them, see https://github.com/EFForg/https-everywhere/pull/10944#issuecomment-313884041